### PR TITLE
Update: oklch()

### PR DESCRIPTION
**Pull request by estelle**
Similar to https://github.com/mdn/content/pull/33007
part of https://github.com/openwebdocs/project/issues/194

### CODE REVIEW

File: files/en-us/web/css/color_value/oklch/index.md [hunk 1]
```diff
@@ -137,68 +159,290 @@ oklch(from hsl(0 100% 50%) calc(l + 0.2) calc(c + 0.1) calc(h - 20) / calc(alpha
...
+
+<div data-color="green"></div>
+<div data-color="green-chroma1"></div>
+<div data-color="green-chroma2"></div>
+<div data-color="green-chroma3"></div>
+```
+
+#### CSS
+
+With the initial starting colors blue, red, and green, we declare progressively smaller values for chroma on them: starting from full color saturation at the highest value of `150` (equivalent to `100%`) down to `3` (equivalent to `2%`), which is almost grey for all the colors.
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/oklch/index.md:263]:
```suggestion
With the initial starting colors blue, red, and green, we declare progressively smaller values for chroma on them: starting from full color saturation at the highest value of `0.4` (equivalent to `100%`) down to `0.01` (equivalent to `2%`), which is almost grey for all the colors.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @estelle. The page is looking good. A few updates are needed.

File: files/en-us/web/css/color_value/oklch/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/color_value/oklch/index.md:263]: good catch!

File: files/en-us/web/css/color_value/oklch/index.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ spec-urls:
 
 {{CSSRef}}
 
-The **`oklch()`** functional notation expresses a given color in the Oklch {{glossary("color space")}}. It has the same L axis as {{cssxref("color_value/oklab","oklab()")}}, but uses polar coordinates C (Chroma) and H (Hue).
+The **`oklch()`** functional notation expresses a given color in the Oklch {{glossary("color space")}}.
+
+OKlch is the cylindrical form of {{CSSXref("color_value/oklab", "OKlab")}}. It uses the same `L` axis as the {{cssxref("color_value/oklab","oklab()")}} color function of the [CIELab color space](/en-US/docs/Glossary/Color_space#cielab_color_spaces), but uses polar coordinates `C` (Chroma) and `H` (Hue).
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/color_value/oklch/index.md]: rewrote the whole thing to:

> The **`oklch()`** functional notation expresses a given color in the Oklab {{glossary("color space")}}. `oklch()` is the cylindrical form of {{CSSXref("color_value/oklab", "OKlab()")}}, using the same `L` axis, and polar Chroma (`C`) and Hue (`h`) coordinates.

File: files/en-us/web/css/color_value/oklch/index.md [hunk 3]
```diff
@@ -39,16 +45,23 @@ oklch(L C H[ / A])
 The parameters are as follows:
 
 - `L`
-  - : A {{CSSXref("&lt;number&gt;")}} between `0` and `1`, a {{CSSXref("&lt;percentage&gt;")}} between `0%` and `100%`, or the keyword `none` (equivalent to `0%` in this case). This value specifies the color's perceived lightness. In this case, the number `0` corresponds to `0%` (black) and the number `1` corresponds to `100%` (white).
+
+  - : A {{CSSXref("&lt;number&gt;")}} between `0` and `1`, a {{CSSXref("&lt;percentage&gt;")}} between `0%` and `100%`, or the keyword `none` (equivalent to `0%` in this case). This value specifies the color's perceived lightness, or "brightness". In this case, the number `0` corresponds to `0%` (black) and the number `1` corresponds to `100%` (white).
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/color_value/oklch/index.md]: moved the lightness /brightness sentence to be last in the paragraph

**Review by estelle — COMMENTED** [review]
Thanks for the review @dipikabh. especially for catching the 150 v. 0.4. good catch!

**Review by dipikabh — APPROVED** [review]
Thanks, @estelle! Just a few minor nits but it is ready to go. Leaving my +1.

File: files/en-us/web/css/color_value/oklch/index.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ spec-urls:
 
 {{CSSRef}}
 
-The **`oklch()`** functional notation expresses a given color in the Oklch {{glossary("color space")}}. It has the same L axis as {{cssxref("color_value/oklab","oklab()")}}, but uses polar coordinates C (Chroma) and H (Hue).
+The **`oklch()`** functional notation expresses a given color in the Oklab {{glossary("color space")}}. `oklch()` is the cylindrical form of {{CSSXref("color_value/oklab", "oklab()")}}, using the same `L` axis, but with polar Chroma (`C`) and Hue (`h`) coordinates.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/color_value/oklch/index.md:13]: the color space is OKlab. OKlch is the color function. 

I finally figured it out when i was writing the "color values" guide. See https://pr33071.content.dev.mdn.mozit.cloud/en-US/docs/Web/CSS/CSS_colors/Color_values#lab_and_oklab. The color spaces we generally will write about are `srgb`, `CIE L*a*b`, and `oklch`. 

- lch() and lab() are in "CIE Lab."
- oklch() and oklab() are in "oklch"

Unless I am wrong again.

**Pull request merged**